pkg/logging/googlepubsub/root.go

```diff
@@ -0,0 +1,28 @@
+package googlepubsub
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/config"
+)
+
+// RootCommand is the parent command for all subcommands in this package.
+// It should be installed under the primary root command.
+type RootCommand struct {
+	common.Base
+	// no flags
+}
+
+// NewRootCommand returns a new command registered in the parent.
+func NewRootCommand(parent common.Registerer, globals *config.Data) *RootCommand {
+	var c RootCommand
+	c.Globals = globals
+	c.CmdClause = parent.Command("googlepubsub", "Manipulate Fastly service version Google Cloud Pub/Sub logging endpoints")
```

Comment:
The API endpoint is called `pubsub`, however that is a bit too generic for my liking, so I'm happy to keep as `googlepubsub`. But noting the inconsistency we may introduce if we do.

Reply:
@phamann yeah, I dislike the inconsistency with go-fastly. But I think we are consistent in the CLI in regards to generic, names (e.g., "blobstorage" in go-fastly is "azureblob" in the CLI).

One of the reasons why I think it is important to disambiguate is because I could see us adding another logging endpoint with "blob storage" or "pub sub" in the name in the future.

Reply:
I do think if we go the route of being consistent with go-fastly in regards to googlepubsub, we should do the same for azureblob.

Reply:
I am going to leave as googlepubsub to remain consistent in the CLI.